app/gui/language/ast/impl/src/id_map.rs

```diff
@@ -0,0 +1,101 @@
+//! A module containing structures describing id-map.
```

Comment:
Might be good to have more explanation about what the ID-map is and what it is meant to be used for.